packages/manager/apps/dedicated/client/app/managedBaremetal/users/activeDirectories/delete/index.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'managedBaremetalUsersFederationDeleteLazyloading';
```

Comment:
same as here :
https://github.com/ovh/manager/pull/8107/files#r986777114

Reply:
It's applied.